.dependabot/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 1
+update_configs:
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "daily"
+    ignored_updates:
+      - match:
+          # Ignore k8s.io dependencies
+          dependency_name: "k8s.io/*"
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "daily"
+    target_branch: "legacy"
```

Comment:
Same as above but only `legacy` applied on this.

Reply:
What happens when we do not have go modules in the legacy branch?

Reply:
Nice catch, I didn't know that it didn't have go modules, I thought it was done by last hackathon who took this over

Reply:
There is a dep option but it doesn't automatically update the `vendor` folder yet. There is an open issue for this

Reply:
For now I'd avoid using this on dep repos/branches